frontend/src/components/Post.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import "./Post.scss";
+
+export default function Post() {
+  return(
+    <div class="post">
+      <div class="image-wrapper">
+        <img src="ig-frontend\src\images\image-icon.png" alt="insert post here"/>
+      </div>
+      <div class="text-wrapper">
+        {/* a Description component can be created to facilitate the creation of Post components */}
+        <div class="post-description">Insert description, timestamp, and poster info here</div>
+        <div class="post-comments">
+          {/* similarly, a Comment component should be later created and implemented here */}
+          <p>comment</p>
```

Comment:
This seems like just a placeholder right? Do we need to push it to master?

Reply:
I think we can for the moment, seeing as this is an initial commit meant more geared towards an easy set-up you guys can replicate and understand when it comes to React functional components. The reason I put so many placeholders was so that you could clearly see the SCSS `overflow-y: auto` in "action".